app/Presenters/LicensePresenter.php

```diff
@@ -36,6 +36,12 @@ public static function dataTableLayout()
                 'switchable' => false,
                 'title' => trans('general.name'),
                 'formatter' => 'licensesLinkFormatter',
+            ], [
+            ], [
+                'field' => 'version',
+                'searchable' => false,
+                'sortable' => true,
```

Comment:
I think we'd need to add `version` to the `$allowed_columns` in the Licenses API in order to sort on it.